src/InfluxDB/Point.php

```diff
@@ -81,26 +88,39 @@ public function __toString()
         return $string;
     }
 
+    /**
+     * @param  array $arr
+     * @return string
+     */
     private function arrayToString(array $arr)
     {
         $strParts = array();
 
         foreach ($arr as $key => $value) {
-            $strParts[] = "{$key}={$value}";
+            $strParts[] = sprintf('%s=%s', $key, $value);
         }
 
-        return implode(",", $strParts);
+        return implode(',', $strParts);
     }
 
     /**
-     * @param $timestamp
-     *
+     * @param  int $timestamp
      * @return bool
      */
     private function isValidTimeStamp($timestamp)
     {
-        return ((int) $timestamp === $timestamp)
-        && ($timestamp <= PHP_INT_MAX)
-        && ($timestamp >= ~PHP_INT_MAX);
+        if ((int) $timestamp === $timestamp) {
```

Comment:
Yes you are correct, it is important to have a good readability. Although i think it's needed in ResultSet.php more than in this method. I think it's good to have this change in there.